zmq-sys/build.rs

```diff
@@ -0,0 +1,34 @@
+extern crate metadeps;
+
+use std::env;
+#[cfg(windows)]
+use std::fs;
+use std::path::Path;
+
+fn prefix_dir(dir: &str) -> Option<String> {
+    env::var("CARGO_MANIFEST_DIR").ok()
+        .map(|prefix| Path::new(&prefix).join("vendor").join("zmq").join(dir))
+        .and_then(|path| path.to_str().map(|p| p.to_owned()))
+}
+
+fn main() {
+    println!("cargo:rerun-if-env-changed=CARGO_MANIFEST_DIR");
+
+    #[cfg(windows)]
+    // hack to get the libzmq dll on the PATH
+    // copies it next to cargo
+    // kind of messy, is less than 1MB
```

Comment:
:+1: I like this as an interim solution until we get static linking working... since this is where cargo install will put the binaries in any case.